### PR TITLE
ERF-77/create zoom controls

### PR DESCRIPTION
**Pull request by lucasoeth**
in this PR I have added a
- custom styled map controls
- react-device-detect

unfortunately, both testing and storybook is not possible because this component only works wrapped inside the map component

### CODE REVIEW

**Review by vogelino — APPROVED** [review]
Great! Well done! Except that you could add a Storybook Story, I've nothing to add! 👌

File: src/components/Map/index.tsx [hunk 1]
```diff
@@ -44,6 +50,7 @@ export const Map: FC<ViewportType> = ({
         setViewport(nextViewport)
       }
     >
+      <MapControls isMobile={isMobile} />
```
Comment (dnsos) [inline src/components/Map/index.tsx:53]: I would argue that we should not add the controls into the map by default. Perhaps in the future we want a map without controls. Instead I would suggest to add the controls in the `RefreshmentMap`, ideally utilizing the `FullScreenMapWrapper` component and its corner slots.

File: src/components/MapControls/index.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react'
+import { NavigationControl, GeolocateControl } from 'react-map-gl'
+
+export const MapControls: FC<{
+  isMobile: boolean
+}> = ({ isMobile }) => {
+  return (
+    <div className={`absolute right-4 ${isMobile ? 'top-4' : 'bottom-4'}`}>
```
Comment (dnsos) [inline src/components/MapControls/index.tsx:8]: Similar to my other comment, I would suggest to leave the positioning of the controls to the parent in which they are rendered.

**Review by dnsos — CHANGES_REQUESTED** [review]
Nice work, @lucasoeth, the controls work and look great.

I would just ask to change the positioning approach for the controls, see my comments for details.

**Review by dnsos — APPROVED** [review]

**Pull request merged**